packages/core/components/Render/index.tsx

```diff
@@ -4,9 +4,13 @@ import { rootDroppableId } from "../../lib/root-droppable-id";
 import { Config, Data } from "../../types/Config";
 import { DropZone, DropZoneProvider } from "../DropZone";
 
-export function Render<
-  UserConfig extends Config<any, any, any> = Config<any, any, any>
->({ config, data }: { config: UserConfig; data: Data }) {
+export function Render<UserConfig extends Config = Config>({
```

Comment:
These types are very deliberate, and I have found that removing the generics causes unexpected issues.

Fixed [here](https://github.com/measuredco/puck/pull/257/commits/82ed63a737e3f9ace114b1a287144ce639776dc7) as part of #257.

Reply:
The problem is it widens the type (effectively returning any) which is probably what fixed your issue, but it means that issue below snuck in because it wasn't strongly typed any more.